cli/tests/testdata/run/internal_dynamic_import.ts.out

```diff
@@ -0,0 +1,4 @@
+error: Uncaught (in promise) TypeError: Cannot load internal module from external code
```

Comment:
How does CLI get to the point of this error? This error is thrown by the internal module loader, not the cli loader. What is up with that?

Reply:
That's a good point @crowlKats, we shouldn't get an error from the CLI loader, not the internal loader.

Reply:
Why was this resolved? AFAICT, the error is still coming from the internal loader. Why is the internal loader used at runtime in CLI?

Reply:
@lucacasonato it's not coming from internal loader - I disabled dynamic imports of internal modules completely in `core/bindings.rs` - so we're not even hitting module system, nor any of the loaders.

Reply:
Can we do the same for resolution in static loading?

Reply:
static loading is erroring before currently, because the FileFetcher is erroring with invalid scheme

Reply:
I think we could do that somewhere in `core/modules.rs` - essentially you're asking to hard disable loading of internal modules in cases where we are running from a snapshot?

Reply:
> essentially you're asking to hard disable loading of internal modules in cases where we are running from a snapshot?

we already are doing that though, no? the InternalModuleLoader isnt used by cli; its not used if one is loading a snapshot but not creating one

Reply:
Please do the enforcement at core level in a follow up as Bartek described. CLI is not the only loader implementation. Not all loaders enforce a given scheme during resolution.